src/main/php/org/bovigo/vfs/vfsStreamAbstractContent.php

```diff
@@ -148,7 +148,7 @@ public function getType()
      * sets the last modification time of the stream content
      *
      * @param   int  $filemtime
-     * @return  vfsStreamContent
+     * @return  $this
```

Comment:
@mikey179 please see http://phpdoc.org/docs/latest/guides/types.html  here u can find $this  ;)